ClosedXML/Excel/Cells/IXLCell.cs

```diff
@@ -310,8 +310,31 @@ public interface IXLCell
 
         IXLCell CopyTo(String target);
 
+        /// <summary>
+        /// Textual representation of cell calculated value (as it is saved to a workbook or read from it)
+        /// </summary>
         String ValueCached { get; }
 
+        /// <summary>
+        /// Calculated value of cell formula. Is used for decreasing number of computations perfromed.
+        /// May hold invalid value when <see cref="RecalculationNeeded"/> flag is True.
+        /// </summary>
+        Object ValueCalculated { get; }
+
+        /// <summary>
+        /// Flag indicating that previously calculated cell value may be not valid anymore and has to be re-evaluated.
+        /// </summary>
+        Boolean RecalculationNeeded { get; }
+
+        /// <summary>
+        /// Perform an evaluation of cell formula. If cell does not contain formula nothing happens, if cell does not need
+        /// recalculation (<see cref="RecalculationNeeded"/> is False) nothing happens either, unless <paramref name="force"/> flag is specified.
+        /// Otherwise recalculation is perfomed, result value is preserved in <see cref="ValueCalculated"/> and returned.
+        /// </summary>
+        /// <param name="force">Flag indicating whether a recalculation must be performed even is cell does not need it.</param>
+        /// <returns>Null if cell does not contain a formula. Calculated value otherwise.</returns>
+        object Evaluate(bool force = false);
```

Comment:
I don't think this property is necessary. I would drop this and rather have a `void InvalidateFormula()` method. Then, only when the cell's value is requested, is the value calculated again. `Evaluate` can stay internal. Agree?